### ISSUE TITLE
The file payload size is inconsistent with the parameter payload size

### ISSUE THREAD

**Issue by HelloRickey** [bnb-chain/greenfield-js-sdk#539]
New problem, I use `delegateUploadObject` to upload json file type will fail, prompt

```
{
  code: '110008',
  message: 'The file payload size is inconsistent with the parameter payload size',
  statusCode: 400
}
```

https://github.com/HelloRickey/greenfield-test/blob/main/test2.js#L35

**Comment by rrr523** [comment]
@HelloRickey 

Thanks for the feedback we have reproduced it, but the reason is not sure, it may be the relationship of SP.

I've tried several common file formats, and `JSON` seems to be the only one that does.

If your needs are urgent, you may need to change the format (such as `txt`), and so on, we will repair the first time.

**Comment by rrr523** [comment]
fixed https://github.com/bnb-chain/greenfield-js-sdk/pull/554.

And released in 2.0.1
